### PR TITLE
CASMINST-6267 Update RELEASE_NOTES.md

### PR DESCRIPTION
**Pull request by mahaluck2005**
# Description

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by heemstra** [comment]
@mahaluck2005 - this needs a few tweaks to remedy the failed automated checks.  For approved/expected word spelling/capitalization, see [this list](https://github.com/Cray-HPE/docs-csm/blob/release/1.4/.spelling).  Also, if needed, words encapsulated by backticks are not checked for spelling.

The markdown style checker is complaining about extra spaces at the end of a few lines.

**Review by rustydb — APPROVED** [review]

**Comment by mahaluck2005** [comment]
@denniswalker Resolved and Ready for Merge.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -27,17 +91,96 @@
...
+* Updated documentation for
+  * CSM upgrade `UPGRADE_KYVERNO_POLICY` step failed due to missing "DVS" namespace
+  * System recovery procedure for Keycloak
+  * Steps to configure SNMP credentials
+  * Use BOS v2 in prepare-images stage
+  * Keycloak to use CMN LB for administrative tasks
+  * Add Cray product catalog module`scripts/operations/configuration/python_lib`
+  * Add NCN `squashfs` IMS ID/version/name to Cray product catalog
+  * Keycloak API upgrade
+  * new name for management NCN CFS configuration
```
Comment (nrockershousen) [inline RELEASE_NOTES.md:129]:
```suggestion
  * New name for management NCN CFS configuration
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -4,19 +4,83 @@
...
+* Singular Method for NCN image customization
+* Updates SLES SP4 Base Image
 
 ### Monitoring
 
-### Networking
+* Implemented pod monitors to scrape SMF Kafka server and zookeeper Prometheus metrics
+* Created  dashboards for Kyverno and to monitor Kyverno policy metrics with Prometheus
+* Created Grafana dashboards to monitor the internals of SMF Kafka server and zookeeper
+* Created Grafana  dashboard for `OpenSearch` cluster monitoring using Prometheus metrics
```
Comment (nrockershousen) [inline RELEASE_NOTES.md:18]:
```suggestion
* Created Grafana dashboard for `OpenSearch` cluster monitoring using Prometheus metrics
```

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**